src/NUnitTestAdapterTests/NavigationDataTests.cs

```diff
@@ -8,47 +8,42 @@ public class NavigationDataTests
         private static readonly string ThisAssemblyPath =
             Path.GetFullPath("NUnit.VisualStudio.TestAdapter.Tests.dll");
 
-        TestConverter testConverter;
+        NavigationDataProvider _provider;
 
         const string Prefix = "NUnit.VisualStudio.TestAdapter.Tests.NavigationTestData";
 
         [SetUp]
         public void SetUp()
         {
-            testConverter = new TestConverter(new TestLogger(), ThisAssemblyPath);
+            _provider = new NavigationDataProvider(ThisAssemblyPath);
         }
 
-        [TearDown]
-        public void TearDown()
-        {
-            testConverter.Dispose();
-        }
-
-        [TestCase("", "EmptyMethod_OneLine", 9, 9, 9, 9)]
-        [TestCase("", "EmptyMethod_TwoLines", 12, 13, 13, 13)]
-        [TestCase("", "EmptyMethod_ThreeLines", 16, 17, 17, 17)]
-        [TestCase("", "EmptyMethod_LotsOfLines", 20, 23, 23, 23)]
-        [TestCase("", "SimpleMethod_Void_NoArgs", 26, 27, 29, 29)]
-        [TestCase("", "SimpleMethod_Void_OneArg", 32, 33, 35, 35)]
-        [TestCase("", "SimpleMethod_Void_TwoArgs", 38, 39, 41, 41)]
-        [TestCase("", "SimpleMethod_ReturnsInt_NoArgs", 44, 45, 46, 47)]
-        [TestCase("", "SimpleMethod_ReturnsString_OneArg", 50, 51, 51, 52)]
+        [TestCase("", "EmptyMethod_OneLine", 9, 9)]
+        [TestCase("", "EmptyMethod_TwoLines", 12, 13)]
+        [TestCase("", "EmptyMethod_ThreeLines", 16, 17)]
+        [TestCase("", "EmptyMethod_LotsOfLines", 20, 23)]
+        [TestCase("", "SimpleMethod_Void_NoArgs", 26, 28)]
+        [TestCase("", "SimpleMethod_Void_OneArg", 32, 33)]
+        [TestCase("", "SimpleMethod_Void_TwoArgs", 38, 39)]
+        [TestCase("", "SimpleMethod_ReturnsInt_NoArgs", 44, 46)]
+        [TestCase("", "SimpleMethod_ReturnsString_OneArg", 50, 51)]
         // Generic method uses simple name
-        [TestCase("", "GenericMethod_ReturnsString_OneArg", 55, 56, 56, 57)]
-        [TestCase("", "AsyncMethod_Void", 60, 61, 64, 64)]
-        [TestCase("", "AsyncMethod_Task", 67, 68, 71, 71)]
-        [TestCase("", "AsyncMethod_ReturnsInt", 74, 75, 78, 78)]
-        [TestCase("+NestedClass", "SimpleMethod_Void_NoArgs", 83, 84, 86, 86)]
-        [TestCase("+ParameterizedFixture", "SimpleMethod_ReturnsString_OneArg", 101, 102, 102, 103)]
+        [TestCase("", "GenericMethod_ReturnsString_OneArg", 55, 56)]
+        [TestCase("", "AsyncMethod_Void", 60, 62)]
```

Comment:
Curious that the release line is +2 for some methods now. I assume navigating to source in VS still works as expected?